core/src/test/scala/cats/effect/ConsoleSpec.scala

```diff
@@ -16,49 +16,50 @@
 
 package cats.effect
 
+import cats.data.EitherT
 import cats.effect.concurrent.Ref
 import cats.instances.boolean._
 import cats.instances.double._
 import cats.instances.int._
 import cats.syntax.all._
-import cats.{Monad, Show}
+import cats.{Applicative, FlatMap, Show}
 import org.scalatest.FunSuite
 
 class ConsoleSpec extends FunSuite {
 
-  case class TestConsole(
-      out1: Ref[IO, List[String]],
-      out2: Ref[IO, List[String]],
-      out3: Ref[IO, List[String]]
-  ) extends Console[IO] {
-    override def putStrLn[A: Show](a: A): IO[Unit] =
+  case class TestConsole[F[_]: Applicative](
+      out1: Ref[F, List[String]],
+      out2: Ref[F, List[String]],
+      out3: Ref[F, List[String]]
+  ) extends Console[F] {
+    override def putStrLn[A: Show](a: A): F[Unit] =
       out1.update(acc => acc ::: a.show :: Nil)
-    override def putStr[A: Show](a: A): IO[Unit] =
+    override def putStr[A: Show](a: A): F[Unit] =
       out2.update(acc => acc ::: a.show :: Nil)
-    override def putError[A: Show](a: A): IO[Unit] =
+    override def putError[A: Show](a: A): F[Unit] =
       out3.update(acc => acc ::: a.show :: Nil)
-    override def readLn: IO[String] = IO.pure("test")
+    override def readLn: F[String] = Applicative[F].pure("test")
```

Comment:
Nitpick: since we are already import all the syntax I'd rather do `"test".pure[F]` :)